cmd/icingadb/main.go

```diff
@@ -104,19 +108,24 @@ func run() int {
 	// the heartbeat is not read while HA gets stuck when updating the instance table.
 	var heartbeat *icingaredis.Heartbeat
 	var ha *icingadb.HA
+	var stats *telemetry.Stats
 	{
 		rc, err := cmd.Redis(logs.GetChildLogger("redis"))
 		if err != nil {
 			logger.Fatalf("%+v", errors.Wrap(err, "can't create Redis client from config"))
 		}
 		heartbeat = icingaredis.NewHeartbeat(ctx, rc, logs.GetChildLogger("heartbeat"))
 
-		db, err := cmd.Database(logs.GetChildLogger("database"))
+		db, err := cmd.Database(logs.GetChildLogger("database"), dbErr)
 		if err != nil {
 			logger.Fatalf("%+v", errors.Wrap(err, "can't create database connection pool from config"))
 		}
 		defer db.Close()
 		ha = icingadb.NewHA(ctx, db, heartbeat, logs.GetChildLogger("high-availability"))
+
+		telemetryLogger := logs.GetChildLogger("telemetry")
```

Comment:
New logger should also be added to docs and example config.

Reply:
I have it on my TODO list..